frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -63,29 +64,40 @@ export function InsightTimeoutState({
                         <h2>Your query took too long to complete</h2>
                     </>
                 )}
-                {isLoading && suggestedSamplingPercentage ? (
-                    <div>
-                        <LemonButton
-                            className="mx-auto mt-4"
-                            type="primary"
-                            onClick={() => {
-                                setSamplingPercentage(suggestedSamplingPercentage)
-                                posthog.capture('sampling_enabled_on_slow_query', {
-                                    samplingPercentage: suggestedSamplingPercentage,
-                                })
-                            }}
-                        >
-                            Click here to speed up calculation with {suggestedSamplingPercentage}% sampling
-                        </LemonButton>
-                        <br />
+                <p className="mx-auto text-center mb-6">Crunching through hogloads of data...</p>
+                <div className="p-4 rounded-lg bg-mid flex gap-x-2">
+                    <div className="flex">
+                        <IconInfo className="w-4 h-4" />
                     </div>
+                    <p className="text-xs m-0">
+                        Need to speed things up? Try reducing the date range{suggestedSamplingPercentage ? ',' : ' or'}{' '}
+                        removing breakdowns
+                        {isLoading && suggestedSamplingPercentage && (
+                            <>
+                                , or turning on sampling with{' '}
+                                <Tooltip
+                                    title="Turning on fast mode will automatically enable 10% sampling for all insights you refresh, speeding up the calculation of results"
+                                    placement="bottom"
+                                >
+                                    <Link
+                                        onClick={() => {
+                                            setSamplingPercentage(suggestedSamplingPercentage)
+                                            posthog.capture('sampling_enabled_on_slow_query', {
+                                                samplingPercentage: suggestedSamplingPercentage,
+                                            })
+                                        }}
+                                    >
+                                        <ThunderboltFilled className="mt-1" /> Fast mode
```

Comment:
We should show the suggestedSamplingPercentage here, as if you click on '10%' and it's still slow this will suggest 1%

Reply:
Ohhh gotcha. Updated with some better state handling!